https_client.cpp

```diff
@@ -1,20 +1,20 @@
 #ifdef _WIN32
-# include <SDKDDKVer.h>
+#include <SDKDDKVer.h>
 #endif
 
 #include <boost/asio.hpp>
 
 #ifdef _WIN32
-# include <boost/asio/windows_sspi.hpp>
+#include <boost/asio/windows_sspi.hpp>
```

Comment:
you really should use your own directory for these files, not boost/asio. this will avoid problems later

Reply:
Will do. I just copied this from the [gnutls implementation](https://github.com/paullouisageneau/boost-asio-gnutls) that I originally tried to mimic.

Should I just use something like `boost-asio-winsspi` as the root of the project/include dir?

Reply:
I would use
```
#include <boost/windows_sspi/windows_sspi.hpp>
```

In other words, the name of your library is Boost.WindowsSSPI

Reply:
OK. Thanks. I'll do a `git move` :-)